demisto_sdk/commands/lint/tests/test_linter/docker_runner_test.py

```diff
@@ -65,19 +70,156 @@ def test_run_pylint_no_errors(self, mocker, linter_obj: Linter):
                                         (8, "test", 0, ""),
                                         (16, "test", 0, ""),
                                         (32, "test", 2, "")])
-    def test_run_pylint_with_errors(self, mocker, linter_obj: Linter, exp_container_exit_code: int, exp_container_log: str,
-                                    exp_exit_code: int, exp_output: str):
+    def test_run_lint_with_errors(self, mocker, linter_obj: Linter, exp_container_exit_code: int, exp_container_log: str,
+                                  exp_exit_code: int, exp_output: str):
         # Docker client mocking
         mocker.patch.object(linter_obj, '_docker_client')
         linter_obj._docker_client.containers.run().wait.return_value = {"StatusCode": exp_container_exit_code}
         linter_obj._docker_client.containers.run().logs.return_value = exp_container_log.encode('utf-8')
-        act_exit_code, act_output = linter_obj._docker_run_pylint(test_image='test-image',
-                                                                  keep_container=False)
+        act_exit_code, act_output = linter_obj._run_linters_in_docker(test_name=self.get_lint_name(),
+                                                                      test_image='test-image',
+                                                                      test_command='test-command',
+                                                                      keep_container=False)
 
         assert act_exit_code == exp_exit_code
         assert act_output == exp_output
 
 
+class TestPylint(BaseLintTest):
+    def get_lint_name(self):
+        return 'Pylint'
+
+
+class TestMypy(BaseLintTest):
+    """
+        Run mypy in python 3 files in docker container
+            - Ensure mypy run in docker container
+            - Ensure the PIP_QUIET exist as env variable in the docker container
+            - Ensure the right command was prepared (the output redirected to /dev/null)
+    """
+
+    def get_lint_name(self):
+        return 'Mypy'
+
+    def test_env_variable_passed_to_docker(self, mocker, linter_obj):
+        """
+        Given: -
+
+        When: - Run Mypy in docker container
+
+        Then: - Validate the environment passed to the run method of the docker container
+
+        """
```

Comment:
If I understand it correctly this is a helper function, so you should have a "regular" docstring and not a test docstring.

Reply:
not a helper function